src/App.js

```diff
@@ -36,51 +39,32 @@ class App extends Component {
           mainGenres: genres.genres1811
         })
       })
-      .then(() => this.getFilters())
       .catch(error => {
         throw new Error(error)
       })
   }
-  getFilters = () => {
+  getFilters = (key) => {
```

Comment:
Line 49: function getFilters() I like how you added "key" as the param, allowing for re-usability of the function when we call the platform and genre searches.